src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  @include displayFlex(column, center);
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid $White-accent;
+  box-sizing: border-box;
+  @include transition(300ms);
```

Comment:
It is a good practice to make transition duration the same for the whole page for better UX